python/vosk/__init__.py

```diff
@@ -47,15 +47,15 @@ def __del__(self):
 
 class KaldiRecognizer(object):
 
-    def __init__(self, *args):
-        if len(args) == 2:
+    def __init__(self, stream, sample_rate, model=None):
```

Comment:
What is stream?

Reply:
It's name of one of arguments, I can change it to something else if you want